polkadot-parachains/tests/purge_chain_works.rs

```diff
@@ -58,7 +58,7 @@ fn purge_chain_works() {
 		let base_path = run_node_and_stop();
 
 		assert!(base_path.path().join("chains/local_testnet/db").exists());
-		assert!(base_path.path().join("polkadot/chains/westend_dev/db").exists());
+		assert!(base_path.path().join("polkadot/chains/westend2/db").exists());
```

Comment:
for the test to succeed we need to change it to westend2, I expect this is related to some westend restart or something but I'm not sure.

Reply:
Ty for fixing it :)